packages/completer/src/handler.ts

```diff
@@ -51,6 +51,7 @@ export class CompletionHandler implements IDisposable {
 
   /**
    * The data connector used to populate completion requests.
+   * @deprecated use `CompletionHandler.ICompletionItesConnector` instead of `IDataConnector`
```

Comment:
```suggestion
   * @deprecated will be removed, or return `CompletionHandler.ICompletionItemsConnector` instead of `IDataConnector` in future versions
```